packages/react-google-maps-api/.eslintrc.js

```diff
@@ -63,6 +64,27 @@ module.exports = {
     }
   },
   rules: {
+    '@getify/proper-arrows/params': [
+      'error', {
+        'unused': 'trailing'
+      }
+    ],
+    '@getify/proper-arrows/name': [
```

Comment:
Is this one really neccesary? or the return one?

> "name": requires => arrow functions to only be used in positions where they receive an inferred name (i.e., assigned to a variable or property, etc), to avoid the poor readbility/debuggability of anonymous function expressions.

I have never seen a codebase that gives names to anonymous arrow functions...

Reply:
well, I think at some point we will be moving to hooks, and there will be a lot of anonymous functions. I already have all Components refactored to functional in my other project, so I think it is safe to make functional version, and compare bundles on size and performance.

Reply:
That is for debuggability - The guy who made this eslint plugin knows what he is doing, but there is no react support out of the box. Maybe we can disable that rule. Well, the thing is that rule asks to refactor () => {} to function functionName() {}

Reply:
In general this rule will help less advanced developers to read the code too.

Reply:
Yeah i read the motivation behind the rule 😊 it just feels a bit harsh... I've never seen code that gives names to anonymous function. I have no principal objection to it though